Makefile

```diff
@@ -75,6 +75,37 @@ dist/nexctl-%: $(NEXCTL_DEPS) | dist
 	$(CMD_PREFIX) CGO_ENABLED=0 GOOS=$(word 2,$(subst -, ,$(basename $@))) GOARCH=$(word 3,$(subst -, ,$(basename $@))) \
 		go build -gcflags="$(NEXODUS_GCFLAGS)" -ldflags="$(NEXODUS_LDFLAGS)" -o $@ ./cmd/nexctl
 
+dist/packages: \
+	dist/packages/linux-amd64 \
+	dist/packages/linux-amd64 \
+	dist/packages/linux-arm \
+	dist/packages/linux-arm64 \
+	dist/packages/darwin-amd64 \
+	dist/packages/darwin-arm64 \
+	dist/packages/windows-amd64
+
+dist/packages/%:
+
+	$(CMD_PREFIX) mkdir -p $@
+	$(ECHO_PREFIX) printf "  %-12s $@/nexd\n" "[GO BUILD]"
+	$(CMD_PREFIX) CGO_ENABLED=0 GOOS=$(word 1,$(subst -, ,$(shell basename $@))) GOARCH=$(word 2,$(subst -, ,$(shell basename $@))) \
+		go build -gcflags="$(NEXODUS_GCFLAGS)" -ldflags="$(NEXODUS_LDFLAGS)" -o $@/nexd$(if $(findstring windows,$@),.exe,) ./cmd/nexd
+	$(ECHO_PREFIX) printf "  %-12s $@/nexctl\n" "[GO BUILD]"
+	$(CMD_PREFIX) CGO_ENABLED=0 GOOS=$(word 1,$(subst -, ,$(shell basename $@))) GOARCH=$(word 2,$(subst -, ,$(shell basename $@))) \
+		go build -gcflags="$(NEXODUS_GCFLAGS)" -ldflags="$(NEXODUS_LDFLAGS)" -o $@/nexctl$(if $(findstring windows,$@),.exe,) ./cmd/nexctl
```

Comment:
instead of duplicating the build commands, you could have this target depend on the corresponding dist/nexd and dist/nexctl targets. The commands here would just be a copy.

I can give it a try and send you a patch

Reply:
not sure how to do that given this target is using %

Reply:
did you see the PR i sent you?  I linked it 2 comments up.

Reply:
I had not.. merged.  thx!